package.json

```diff
@@ -56,6 +56,7 @@
     "eslint-plugin-cypress": "2.11.2",
     "eslint-plugin-react": "7.23.2",
     "exports-loader": "2.0.0",
+    "fishery": "1.4.0",
```

Comment:
@ClementChaumel @albertkol I picked this factory lib as it supports js as well as ts so we can start using it for the js React files and then if they get migrated to ts the factories will already be there.